### PR TITLE
Feature/DACT-956 CH01 feature flag and button

### PR DESCRIPTION
**Pull request by mdesaikns**
* added CH01 flag for a web.
* conditionally display the button.
* prototype https://add-and-remove-dev-ready-f0f9fa3155cc.herokuapp.com/directors

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars [hunk 1]
```diff
@@ -4,6 +4,7 @@ aws_profile = "staging-eu-west-2"
 account_local_url = "https://staging.company-information.service.gov.uk"
 feature_flag_wle = "false"
 feature_flag_ap01_web = "true"
+feature_flag_ch01_web = "true"
```
Comment (rwalrondch) [inline terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars:7]: Can we set this to be false in the first instance please. We don't want any CH01 features being in there whilst we are doing pen testing etc.

**Review by rwalrondch — APPROVED** [review]

File: src/controllers/start.controller.ts [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+  CH01_ACTIVE
+} from "../utils/properties";
 import { Templates } from "../types/template.paths";
 
 export const get = (req: Request, res: Response) => {
   return res.render(Templates.START, { CHS_URL,
     PIWIK_START_GOAL_ID,
     FEATURE_FLAG_REMOVE_DIRECTOR_20022023,
     AP01_ACTIVE,
+    CH01_ACTIVE,
```
Comment (cdrainkainos) [inline src/controllers/start.controller.ts:17]: Do we need this here? Might be left over from testing

Comment (mdesaikns) [reply src/controllers/start.controller.ts:17]: yes I did noticed that, however I believe it's going to be useful to display the content of the [start ](https://github.com/companieshouse/officer-filing-web/blob/main/views/start.html#L16-L23) page.

**Review by clewis1 — APPROVED** [review]
Looks good

**Pull request merged**